.github/workflows/coding-standards.yml

```diff
@@ -24,11 +24,8 @@ jobs:
         run: |
           make install -j10 -O
 
-      - name: "install mago"
-        run: wget -qO- https://carthage.software/mago.sh | bash -s -- --install-dir="/usr/local/bin"
```

Comment:
Small downside : since make install takes the locked mago, one needs to bump the dependencies in order te run on newer mago version. It makes the pipeline more stable though ... I'll leave this up to you to decide how to best deal with this: I'dd say a periodic update through PRs is probably the best in terms of compatibility.